pkg/build/api/types.go

```diff
@@ -54,6 +54,12 @@ const (
 	// BuildConfigPausedAnnotation is an annotation that marks a BuildConfig as paused.
 	// New Builds cannot be instantiated from a paused BuildConfig.
 	BuildConfigPausedAnnotation = "openshift.io/build-config.paused"
+	// BuildCompletedAnnotation is an annotation that marks a Build as completed and
+	// will prevent the build controller from processing it further.
+	BuildCompletedAnnotation = "openshift.io/build.completed"
+	// BuildAcceptedAnnotation is an annotation used to update a build that has been
+	// created so it can be seen build controller queue before a resync.
+	BuildAcceptedAnnotation = "openshift.io/build.accepted"
```

Comment:
the new annotation format(sigh) is 
BuildSourceSecretMatchURIAnnotationPrefix = "build.openshift.io/source-secret-match-uri-"

please align these new annotations with that.